### ISSUE TITLE
`KeyError: 'locale'` - Google `userinfo` API not always returning locale

### ISSUE THREAD

**Issue by sfowlr** [megalus/django-google-sso#31]
As of today, the `www.googleapis.com/oauth2/v2/userinfo` API endpoint has started returning a slightly different payload for some users - one that does not include the `locale` key.  The majority of our Enterprise Google accounts can still sign in successfully, but several are getting the following 500 error:

```
Internal Server Error: /google_sso/callback/
Traceback (most recent call last):
  File "/var/www/cnm.cox2m.com/.venv/lib/python3.8/site-packages/asgiref/sync.py", line 534, in thread_handler
    raise exc_info[1]
  File "/var/www/cnm.cox2m.com/.venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 38, in inner
    response = await get_response(request)
  File "/var/www/cnm.cox2m.com/.venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 233, in _get_response_async
    response = await wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/cnm.cox2m.com/.venv/lib/python3.8/site-packages/asgiref/sync.py", line 479, in __call__
    ret: _R = await loop.run_in_executor(
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/var/www/cnm.cox2m.com/.venv/lib/python3.8/site-packages/asgiref/sync.py", line 538, in thread_handler
    return func(*args, **kwargs)
  File "/var/www/cnm.cox2m.com/.venv/lib/python3.8/site-packages/django/views/decorators/http.py", line 40, in inner
    return func(request, *args, **kwargs)
  File "/var/www/cnm.cox2m.com/.venv/lib/python3.8/site-packages/django_google_sso/views.py", line 95, in callback
    user = user_helper.get_or_create_user()
  File "/var/www/cnm.cox2m.com/.venv/lib/python3.8/site-packages/django_google_sso/main.py", line 112, in get_or_create_user
    google_user.locale = self.user_info["locale"]
KeyError: 'locale'
```

I ended up monkeypatching this today as a temporary workaround, changing this line:
https://github.com/megalus/django-google-sso/blob/e4f457e1efcf129ddd7be70bff39572daab605e5/django_google_sso/main.py#L117

to this:
`google_user.locale = self.user_info.get("locale", 'en')`

the 'en' parameter should probably become a configurable setting, i.e. `GOOGLE_SSO_DEFAULT_LOCALE` in the long run.

**Comment by sfowlr** [comment]
Thanks for the quick fix!

**Comment by jkkitakita** [comment]
Hi @chrismaille,

I am using Python 3.9, in which case I need to use v3.3.0.
However, this issue prevents me from using v3.3.0.
I would like to fix v3.3.0. What should I do?
